lib/ndebug/include/lib/ndebug/shared_structs.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+struct __attribute__((__packed__)) ndebug_ctrl_packet_t {
```

Comment:
Will that work when we're building the host side tools since this file is shared between the host tools and the LK build?